test/e2e/common_test.go

```diff
@@ -1209,7 +1209,7 @@ func generatePolicyFile(tempDir string) string {
         }
     }
 }
-`, keyPath)
+`, port, keyPath, port, port)
```

Comment:
FYI, I suggest passing only port, keyPath once and then using `%[1]d` and `%[2]s` as formatter (you can use them multiple times)
https://pkg.go.dev/fmt#hdr-Explicit_argument_indexes

Reply:
TIL! Thank you for that, it's much cleaner now.